x-pack/plugins/observability/public/components/app/ingest_manager_panel/index.tsx

```diff
@@ -40,7 +40,7 @@ export function IngestManagerPanel() {
           </EuiText>
         </EuiFlexItem>
         <EuiFlexItem>
-          <EuiLink href="https://ela.st/ingest-manager-announcement">
+          <EuiLink href="/app/ingestManager#/">
```

Comment:
Shouldn't we use a Kibana service to link? (btw, typo below, we might want to fix that in a separate PR)

Reply:
Which kibana service you're talking about?

Reply:
Something like `core.http.basePath.prepend()`?

Reply:
ah I thought was something else... just added it, and fixed the typo

Reply:
Awesome!